Marlin/Configuration_adv.h

```diff
@@ -202,13 +202,7 @@
 #endif
 
 #if ENABLED(PIDTEMP)
-  // Add an experimental additional term to the heater power, proportional to the extrusion speed.
-  // A well-chosen Kc value should add just enough power to melt the increased material volume.
-  //#define PID_EXTRUSION_SCALING
-  #if ENABLED(PID_EXTRUSION_SCALING)
-    #define DEFAULT_Kc (100) //heating power=Kc*(e_speed)
-    #define LPQ_MAX_LEN 50
-  #endif
```

Comment:
Do not remove PID_EXTRUSION_SCALING.
I just suggested removing  LPQ_MAX_LEN feature. I can't imagine scenario where you want to lag additional heating after extruding has taken place. To me it looks like some remain of workaround where it was not possible to rig directly to stepper.position(E_AXIS) and it was rigged somewhere before planer queue.

    #if ENABLED(PID_EXTRUSION_SCALING)
        /**
         * Increase in PWM duty cycle needed to to extrude 1 mm^2 per second
         * of filament if extruder temperature is 1 Kelvin above ambient
         * temperature [s*K^-1*mm^-2]
         */
        #define DEFAULT_Kc 0.009517f
    #endif